pagerduty/provider.go

```diff
@@ -46,27 +46,28 @@ func Provider() *schema.Provider {
-			"pagerduty_schedule":                  dataSourcePagerDutySchedule(),
-			"pagerduty_user":                      dataSourcePagerDutyUser(),
-			"pagerduty_users":                     dataSourcePagerDutyUsers(),
-			"pagerduty_user_contact_method":       dataSourcePagerDutyUserContactMethod(),
-			"pagerduty_team":                      dataSourcePagerDutyTeam(),
-			"pagerduty_vendor":                    dataSourcePagerDutyVendor(),
-			"pagerduty_extension_schema":          dataSourcePagerDutyExtensionSchema(),
-			"pagerduty_service":                   dataSourcePagerDutyService(),
-			"pagerduty_service_integration":       dataSourcePagerDutyServiceIntegration(),
-			"pagerduty_business_service":          dataSourcePagerDutyBusinessService(),
-			"pagerduty_priority":                  dataSourcePagerDutyPriority(),
-			"pagerduty_ruleset":                   dataSourcePagerDutyRuleset(),
-			"pagerduty_tag":                       dataSourcePagerDutyTag(),
-			"pagerduty_event_orchestration":       dataSourcePagerDutyEventOrchestration(),
-			"pagerduty_event_orchestrations":      dataSourcePagerDutyEventOrchestrations(),
-			"pagerduty_automation_actions_runner": dataSourcePagerDutyAutomationActionsRunner(),
-			"pagerduty_automation_actions_action": dataSourcePagerDutyAutomationActionsAction(),
-			"pagerduty_incident_workflow":         dataSourcePagerDutyIncidentWorkflow(),
-			"pagerduty_custom_field":              dataSourcePagerDutyField(),
-			"pagerduty_custom_field_schema":       dataSourcePagerDutyFieldSchema(),
+			"pagerduty_escalation_policy":               dataSourcePagerDutyEscalationPolicy(),
+			"pagerduty_schedule":                        dataSourcePagerDutySchedule(),
+			"pagerduty_user":                            dataSourcePagerDutyUser(),
+			"pagerduty_users":                           dataSourcePagerDutyUsers(),
+			"pagerduty_user_contact_method":             dataSourcePagerDutyUserContactMethod(),
+			"pagerduty_team":                            dataSourcePagerDutyTeam(),
+			"pagerduty_vendor":                          dataSourcePagerDutyVendor(),
+			"pagerduty_extension_schema":                dataSourcePagerDutyExtensionSchema(),
+			"pagerduty_service":                         dataSourcePagerDutyService(),
+			"pagerduty_service_integration":             dataSourcePagerDutyServiceIntegration(),
+			"pagerduty_business_service":                dataSourcePagerDutyBusinessService(),
+			"pagerduty_priority":                        dataSourcePagerDutyPriority(),
+			"pagerduty_ruleset":                         dataSourcePagerDutyRuleset(),
+			"pagerduty_tag":                             dataSourcePagerDutyTag(),
+			"pagerduty_event_orchestration":             dataSourcePagerDutyEventOrchestration(),
+			"pagerduty_event_orchestrations":            dataSourcePagerDutyEventOrchestrations(),
+			"pagerduty_event_orchestration_integration": dataSourcePagerDutyEventOrchestrationIntegration(),
```

Comment:
Added `pagerduty_event_orchestration_integration` to the list and reformatted the block.